src/components/pages/TeacherDashboard/TeacherDashboardSectionList/TeacherDashboardSectionList.tsx

```diff
@@ -13,28 +16,48 @@ const RenderTeacherDashboardSectionList = ({
   const [newSectionOpen, setNewSectionOpen] = useState(false);
   const openSectionModal = () => setNewSectionOpen(true);
   const openInviteModal = () => setInviteModalOpen(true);
+  const sectionList = useRecoilValue(list);
```

Comment:
sections (props) vs sectionList (recoil) we need to make sure we're using the correct list